code/modules/mob/living/life.dm

```diff
@@ -183,9 +183,6 @@
 			if(LAZYACCESS(chem_doses, T) <= 0)
 				LAZYREMOVE(chem_doses, T)
 
-	if(apply_chemical_effects())
-		updatehealth()
```

Comment:
This... is necessary to make sure that `CE_REGEN_BURN` and `CE_REGEN_BRUTE` cause fewer repeated update_health calls, no? Now `heal_organ_damage()` calls it directly, meaning other overrides will build onto that—for example, downstream I have a couple extra healing chem effects on top of these two. The check is to make sure update_health() is delayed until `apply_chemical_effects()` is over.